late/scripts/chroot.sh

```diff
@@ -81,7 +81,7 @@ if ! mount | grep "$TARGET/run"; then
     MOUNT_CLEANUP="$TARGET/run $MOUNT_CLEANUP"
 fi
 if ! mount | grep "$TARGET/pts"; then
-    if [ ! -e "$TARGET/pts"]; then
```

Comment:
what is wrong with this?

Reply:
closes #129

Reply:
Shouldn't the fix add the space not revert?

Reply:
Ok, have the fix included in commit d543a9836635972868ec0103805c1f9a05c467e8.